apps/console/src/features/authentication-flow-builder/components/script-editor-panel/script-editor-panel.tsx

```diff
@@ -147,15 +195,46 @@ const ScriptEditorPanel = (props: PropsWithChildren<ScriptEditorPanelPropsInterf
         return authenticationSequence?.script;
     };
 
-    return (
-        <Suspense fallback={ <CircularProgress /> }>
-            <div className={ classNames("script-editor-panel", className) } data-componentid={ componentId }>
-                <Box className="toolbar-container">
-                    <Toolbar variant="dense">
-                        <Box>
-                            <Typography>{ t("console:loginFlow.scriptEditor.panelHeader") }</Typography>
-                        </Box>
-                        <div className="actions">
+    /**
+     * Function to handle the script editor panel size change.
+     */
+    const handleScriptEditorPanelSizeChange = (): void => {
+        if (scriptEditorPanelSizeMode === ScriptEditorPanelSizeModes.Minimized) {
+            setScriptEditorPanelSizeMode(ScriptEditorPanelSizeModes.Maximized);
+
+            return;
+        }
+
+        setScriptEditorPanelSizeMode(ScriptEditorPanelSizeModes.Minimized);
+    };
+
+    const ScriptEditor: ReactElement = (
+        <MonacoEditor
+            loading={ null }
+            className="script-editor"
+            width="100%"
+            height="100%"
+            language="javascript"
+            theme={ editorTheme }
+            value={ getAdaptiveScript() }
+            options={ {
+                automaticLayout: true
+            } }
+            onChange={ handleScriptChange }
+            onMount={ handleEditorOnMount }
+            data-componentid={ `${componentId}-code-editor` }
+        />
+    );
+
+    const ScriptEditorToolbar: ReactElement = (
+        <Box className="script-editor-toolbar-container">
+            <Toolbar variant="dense">
+                <Box>
+                    <Typography>{ t("console:loginFlow.scriptEditor.panelHeader") }</Typography>
+                </Box>
+                <div className="actions">
+                    { scriptEditorPanelSizeMode === ScriptEditorPanelSizeModes.Minimized && (
```

Comment:
Secret Dropdown Popover needs to be properly handled for fullscreen mode. Removed the theme dropdown and secret dropdown from full-screen mode for the time being.

<img width="2045" alt="Screenshot 2024-01-28 at 15 14 01" src="https://github.com/wso2/identity-apps/assets/25959096/714a6955-cd55-429b-bbc1-e2275e61f1e0">